main.go

```diff
@@ -280,7 +280,9 @@ func generate(in configInput) ([]byte, error) {
 		if in.includeXDSTPNameInLDS {
 			tdAuthority := "traffic-director-global.xds.googleapis.com"
 			c.Authorities[tdAuthority] = Authority{
-				ClientListenerResourceNameTemplate: fmt.Sprintf("xdstp://%s/envoy.config.listener.v3.Listener/%%s", tdAuthority),
+				// Listener Resource Name format for p2c config looks like:
```

Comment:
Umm...
1. This entire generator isn't used for P2C
2. This is the normal TD case, no directpath involved

Reply:
According to the IYKYK/td-xds-federation document, `traffic-director-global.xds.googleapis.com` authority was in the entry that said P2C config. I was trying to stick to that name to find backref. 

So P2C means DirectPath but in the other direction?

Reply:
Gotcha! Thanks for explaining